doc/source/storage-configuration.rst

```diff
@@ -0,0 +1,79 @@
+.. _storage.config:
+
+Remote backup storage 
+*********************************************************************************
+
+.. contents::
+   :local:
+
+|PBM| supports the following types of remote backup storages:
+
+* S3-compatible storage
+* Filesystem type storage
+
+.. rubric:: S3 compatible storage
+
+|PBM| should work with other S3-compatible storages but was only tested with the following ones:
+
+- `Amazon Simple Storage Service <https://docs.aws.amazon.com/s3/index.html>`_ 
+- `Google Cloud Storage <https://cloud.google.com/storage>`_
+- `MinIO <https://min.io/>`_
+  
+Starting from v1.3.2, |PBM| supports :term:`server-side encryption <Server-side encryption>` for :term:`S3 buckets <Bucket>` with customer managed keys stored in |AWS KMS|.
+
+.. seealso::
+
+   `Protecting Data Using Server-Side Encryption with CMKs Stored in AWS Key Management Service (SSE-KMS) <https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingKMSEncryption.html>`_
+
```

Comment:
I don't think this detailed config option should be mentioned here, in the middle of sections giving an overview of storage configuration at it's highest level

Reply:
It's included in the configuration-options.rst page I see.

If the version information is important I think we should add "New in v1.3.2" to the part ".. rubric:: Server-side encryption options" in configuration-options.rst